### PR TITLE
[Bots][AddedclientInfoentity][1492175]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added get clientInfo in your bot context section with json and c# code. Reviewed document for grammar and formatting. Please review.

### CODE REVIEW

File: msteams-platform/resources/bot-v3/bots-context.md [hunk 1]
```diff
@@ -11,23 +11,24 @@ ms.date: 05/20/2019
...
+> Microsoft Teams&ndash;specific bot APIs are best accessed through our extensions for the Bot Builder SDK. For C# or .NET, download our [Microsoft.Bot.Connector.Teams](https://www.nuget.org/packages/Microsoft.Bot.Connector.Teams) NuGet package. For Node.js development, the BotBuilder for Teams functionality has been incorporated into the [Bot Framework SDK](https://github.com/microsoft/botframework-sdk) as of v4.6.
 
-## Fetching the team roster
+## Fetch the team roster
 
-Your bot can query for the list of team members and their basic profiles, which includes Teams user IDs and Azure Active Directory (Azure AD) information such as name and objectId. You can use this information to correlate user identities; for example, to check whether a user logged into a tab through Azure AD credentials is a member of the team.
+Your bot can query for the list of team members and their basic profiles, which includes Teams user IDs and Azure Active Directory (AAD) information such as name and objectId. You can use this information to correlate user identities. For example, to check whether a user logged into a tab through AAD credentials is a member of the team.
```
Comment (KirtiPereira) [inline msteams-platform/resources/bot-v3/bots-context.md:18]:
```suggestion
Your bot can query for the list of team members and their basic profiles, which includes Teams user IDs and Azure Active Directory (AAD) information such as name and objectId. You can use this information to correlate user identities. For example, to check if a user logged into a tab through AAD credentials is a member of the team.
```

Comment (KirtiPereira) [inline msteams-platform/resources/bot-v3/bots-context.md:18]: Should **objectId** be object ID? Please verify and update.

File: msteams-platform/resources/bot-v3/bots-context.md [hunk 2]
```diff
@@ -11,23 +11,24 @@ ms.date: 05/20/2019
...
 
 ### REST API example
 
-You can directly issue a GET request on [`/conversations/{teamId}/members/`](/bot-framework/rest-api/bot-framework-rest-connector-api-reference#get-conversation-members), using the value of `serviceUrl` as the endpoint.
+You can directly issue a GET request on [`/conversations/{teamId}/members/`](/bot-framework/rest-api/bot-framework-rest-connector-api-reference#get-conversation-members), using the `serviceUrl` value as the endpoint.
```
Comment (KirtiPereira) [inline msteams-platform/resources/bot-v3/bots-context.md:22]:
```suggestion
Directly issue a GET request on [`/conversations/{teamId}/members/`](/bot-framework/rest-api/bot-framework-rest-connector-api-reference#get-conversation-members), using the `serviceUrl` value as the endpoint.
```

File: msteams-platform/resources/bot-v3/bots-context.md [hunk 3]
```diff
@@ -11,23 +11,24 @@ ms.date: 05/20/2019
...
 
 The `teamId` can be found in the `channeldata` object of the activity payload that your bot receives in the following scenarios:
-* When a user messages or interacts with your bot in a team context (see [Receiving Messages](~/resources/bot-v3/bot-conversations/bots-conversations.md#receiving-messages))
-* When a new user or bot is added to a team (see [Bot or user added to a team](~/resources/bot-v3/bots-notifications.md#bot-or-user-added-to-a-team))
 
 > [!NOTE]
->* Make sure to use the team id when calling the api
->* The value of `serviceUrl` tends to be stable but can change. When a new message arrives, your bot should verify its stored value of `serviceUrl`.
+>* Make sure to use the team ID when calling the API.
+>* The value of `serviceUrl` tends to be stable but can change. When a new message arrives, your bot must verify its stored value of `serviceUrl`.
```
Comment (KirtiPereira) [inline msteams-platform/resources/bot-v3/bots-context.md:28]:
```suggestion
>* The value of `serviceUrl` tends to be stable but can change. When a new message arrives, your bot must verify its stored `serviceUrl` value.
```

File: msteams-platform/resources/bot-v3/bots-context.md [hunk 4]
```diff
@@ -124,12 +125,12 @@ Your bot can query the list of channels in a team.
 
 ### REST API example
 
-You can directly issue a GET request on `/teams/{teamId}/conversations/`, using the value of `serviceUrl` as the endpoint.
+You can directly issue a GET request on `/teams/{teamId}/conversations/`, using the `serviceUrl` value as the endpoint.
```
Comment (KirtiPereira) [inline msteams-platform/resources/bot-v3/bots-context.md:128]:
```suggestion
Directly issue a GET request on `/teams/{teamId}/conversations/`, using the `serviceUrl` value as the endpoint.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@v-rodriguezd - please verify and incorporate the suggestion.

**Comment by KirtiPereira** [comment]
Also, please work on the Warnings.

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by Trinetra-MSFT — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**